_format/1.0/index.md

```diff
@@ -926,13 +926,13 @@ If an endpoint supports the `include` parameter and a client supplies it,
 the server **MUST NOT** include unrequested [resource objects] in the `included`
 section of the [compound document].
 
+If a server is unable to identify a relationship path or does not support
+inclusion of resources from a path, it **MUST** respond with 400 Bad Request.
+
```

Comment:
That's actually what I did at first, but then the paragraph starting with "For instance" seems like it applies to the paragraph starting with "If a server is unable to identify a relationship path".  The "For instance" paragraph is the start of examples about `include`ing, so perhaps it should be another section?